backend/migrations/sqls/20241016182736-add-observality-down.sql

```diff
@@ -0,0 +1,26 @@
+DROP TRIGGER IF EXISTS set_timestamp ON metadata;
+DROP TRIGGER IF EXISTS set_timestamp ON nodes;
+DROP TRIGGER IF EXISTS set_timestamp ON transaction_results;
+DROP TRIGGER IF EXISTS set_timestamp ON users;
+DROP TRIGGER IF EXISTS set_timestamp ON object_ownership;
+DROP TRIGGER IF EXISTS set_timestamp ON api_keys;
+
+ALTER TABLE metadata DROP COLUMN created_at;
+ALTER TABLE metadata DROP COLUMN updated_at;
+
+ALTER TABLE nodes DROP COLUMN created_at;
+ALTER TABLE nodes DROP COLUMN updated_at;
+
+ALTER TABLE transaction_results DROP COLUMN created_at;
+ALTER TABLE transaction_results DROP COLUMN updated_at;
+
+ALTER TABLE users DROP COLUMN created_at;
+ALTER TABLE users DROP COLUMN updated_at;
+
+ALTER TABLE object_ownership DROP COLUMN created_at;
+ALTER TABLE object_ownership DROP COLUMN updated_at;
+
+ALTER TABLE api_keys DROP COLUMN created_at;
+ALTER TABLE api_keys DROP COLUMN updated_at;
```

Comment:
last_upload_at, last_download_at could be good to have
even a usage counter

Reply:
Has been outdated since now there's an entry in table `interactions` for each r/w op